### PR TITLE
Enable all DOCS_ options for packaging workbench

### LINKED ISSUE #72: Missing plots from in-built docs
**Describe the bug**

Packages are missing the in-built plots in QtHelp.

Found in mantidproject/mantid#34057.

**To Reproduce**

1. Start workbench
2. Open Help
3. Go to CalculatePlaczekCorrection and scroll down.
4. The image shows not enabled

**Expected behavior**

A plot should be there.

**Screenshots**

image

**Platform/Version (please complete the following information):**
 - OS: All
 - OS Version: 
 - Conda version: Current main
 
**Additional context**

### PR DESCRIPTION
**Pull request by martyngigg**
**Description**

Without this parts of the documentation are not rendered
for users.

**Test**

Review the changes and check the options match the old [buildscript](https://github.com/mantidproject/mantid/blob/main/buildconfig/Jenkins/buildscript#L266)

Fixes #72

### CODE REVIEW

File: recipes/mantidworkbench/bld.bat [hunk 1]
```diff
@@ -8,6 +8,10 @@ cmake ^
     -DCONDA_ENV=True ^
     -DENABLE_PRECOMMIT=OFF ^
     -DENABLE_DOCS=ON ^
+    -DDOCS_DOTDIAGRAMS=ON ^
+    -DDOCS_SCREENSHOTS=ON ^
```
Comment (thomashampson) [inline recipes/mantidworkbench/bld.bat:12]: `-DDOCS_SCREENSHOTS=ON ^` is duplicated below

File: recipes/mantidworkbench/build.sh [hunk 2]
```diff
@@ -39,6 +39,10 @@ cmake \
   -DCMAKE_FIND_FRAMEWORK=LAST \
   -DOpenSSL_ROOT=$PREFIX \
   -DENABLE_DOCS=ON \
+  -DDOCS_DOTDIAGRAMS=ON \
+  -DDOCS_SCREENSHOTS=ON \
```
Comment (thomashampson) [inline recipes/mantidworkbench/build.sh:43]: `-DDOCS_SCREENSHOTS=ON ^` is duplicated below

**Review by thomashampson — CHANGES_REQUESTED** [review]
There are a couple of duplicated lines.

File: recipes/mantidworkbench/bld.bat (hunk 1 above)
Comment (martyngigg) [reply recipes/mantidworkbench/bld.bat:12]: Ah yes! We must have realised that at some point.

**Comment by martyngigg** [comment]
Thanks for catching that. I've removed the duplicate lines.

**Review by thomashampson — APPROVED** [review]

**Pull request merged**